examples/common-switches.mk

```diff
@@ -130,6 +131,10 @@ ifeq ($(ECDSA),1)
 COMMONCFLAGS                   += -DOPENTHREAD_CONFIG_ECDSA_ENABLE=1
 endif
 
+ifeq ($(EXTERNAL_HEAP),1)
+COMMONCFLAGS                   += -DOPENTHREAD_CONFIG_EXTERNAL_HEAP_ENABLE=1
```

Comment:
nit: how about `OPENTHREAD_CONFIG_HEAP_EXTERNAL_ENABLE` for consistency?

Reply:
Sorry, I totally forgot about this one.